CHANGES

```diff
@@ -13304,4 +13309,4 @@ des-cbc           3624.96k     5258.21k     5530.91k     5624.30k     5628.26k
 
   *) A minor bug in ssl/s3_clnt.c where there would always be 4 0
      bytes sent in the client random.
-     [Edward Bishop <ebishop@spyglass.com>]
+     [Edward Bishop <ebishop@spyglass.com>]
```

Comment:
Did your editor eat the final newline here?

Reply:
It must have.  To avoid a possible merge conflict with other commits hitting master, I am going to leave this as is on this last commit.